src/Cli/ConfigGenerator.cs

```diff
@@ -286,6 +311,30 @@ private static bool TryDetermineIfApiIsEnabled(bool apiDisabledOptionValue, CliB
             return true;
         }
 
+        /// <summary>
+        /// Helper method to determine if the nested create operation is enabled or not based on the inputs from dab init command.
+        /// </summary>
+        /// <param name="nestedCreateEnabledOptionValue">Input value for --graphql.nested-create.enabled option of the init command</param>
+        /// <param name="isNestedCreateEnabledForGraphQL">Boolean value indicating if nested create operation is enabled.</param>
+        private static bool IsNestedCreateOperationEnabled(CliBool nestedCreateEnabledOptionValue, out bool isNestedCreateEnabledForGraphQL)
+        {
+            if (nestedCreateEnabledOptionValue is CliBool.None)
+            {
+                isNestedCreateEnabledForGraphQL = false;
```

Comment:
still confused here. Set `isNestedCreateEnabledForGraphQL` to false but return true for `IsNestedCreateOperationEnabled`

Reply:
The return value indicates whether the function ran into issues when determining nested create is enabled or not for graphQL. The reason for doing this is to account for any errors that might arise when parsing the value entered to a boolean value.

But this is unnecessary, because the function takes a CliBool type parameter which can take only 3 values - None, True or False. And, this function is called only when the value is either True/False. So, there should not be any such errors when doing `bool.parse(...)`

So, I've updated the function to directly return the value of whether nested create is enabled or not.